validator/templates/validator/help.html

```diff
@@ -49,7 +49,7 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
     <p>
```

Comment:
I know it was there before, but could you change the first sendetnce: 'Each area has menu with two icons on the top right.' to 'Each area has a menu with two icons on the top right.'. Also, what is the area exactly?

Reply:
I think the area refers to each setting component. It wasn't me who was writing it :)